files/working-hours.sh

```diff
@@ -2,14 +2,97 @@
 
 DRY_RUN=${DRY_RUN:-0}
 NAMESPACES=${NAMESPACES:-}
+PROTECTED_NAMESPACES=${PROTECTED_NAMESPACES:-}
 NAMESPACES_LABEL_SELECTOR=${NAMESPACES_LABEL_SELECTOR:-}
+NAMESPACES_ALL_LABEL_SELECTOR=${NAMESPACES_ALL_LABEL_SELECTOR:-}
 DEPLOYMENTS_LABEL_SELECTOR=${DEPLOYMENTS_LABEL_SELECTOR:-}
+STATEFULSETS_LABEL_SELECTOR=${STATEFULSETS_LABEL_SELECTOR:-}
 GO_TO_REPLICAS=${GO_TO_REPLICAS:-1}
 
+
+filter_namespaces() {
+  # Usage examples:
+  # With warnings (default):
+  # NAMESPACES=$(filter_namespaces "${PROTECTED_NAMESPACES}" "${NAMESPACES}")
+  # Without warnings:
+  # NAMESPACES=$(filter_namespaces "${PROTECTED_NAMESPACES}" "${NAMESPACES}" 1)
+    local protected_namespaces="$1"
+    local namespaces_to_filter="$2"
```

Comment:
```suggestion
    local protected_namespaces="${1}"
    local namespaces_to_filter="${2}"
```

Just for consistency.